CHANGELOG.md

```diff
@@ -3,6 +3,61 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 2022-10-18
```

Comment:
```suggestion
## 2022-10-18 Part 3 - Graduating packages release
```

This needs to change. That's my suggestion. But not 100% sure if it's desirable.

Reply:
I was going to suggest tomorrow as the release date so it doesn't look as bad i.e. `2022-10-19`.